src/Data/Functor/Linear/Internal/Functor.hs

```diff
@@ -17,9 +26,12 @@ import qualified Control.Monad.Trans.Reader as NonLinear
 import qualified Control.Monad.Trans.State.Strict as Strict
 import Data.Functor.Compose
 import Data.Functor.Const
-import Data.Functor.Identity
+import Data.Functor.Identity (Identity (..))
```

Comment:
Is there any naming conflict if we import the whole `Data.Functor.Identity` module unqualified?

Reply:
No; the module doesn't (currently) contain anything else. Even if something is added, though, we're quite unlikely to need it, so I don't see the point in changing this. I'll do it if you want.

Reply:
Well, I don't mind personally, but I think that this is the import style that @aspiwack likes and it's the one that is used (in principle) across the codebase.